pylutron_caseta/__init__.py

```diff
@@ -3,35 +3,6 @@
 # This is the private key for the Lutron Smart Bridge 'leap' user.
 # https://github.com/njschwartz/Lutron-Smart-Pi/blob/master/RaspberryPi/LutronPi.py
```

Comment:
that comment should probably be removed too

Reply:
Makes sense :-)